launch_ros/launch_ros/actions/push_ros_namespace.py

```diff
@@ -29,7 +29,8 @@
 
 from rclpy.validate_namespace import validate_namespace
 
-
+# TODO(ivanpauno): Generalize launch.frontend so both hyphens and underscores are equivalent.
+@expose_action('push_ros_namespace')
```

Comment:
@ivanpauno should this be in its own separate PR? Same for all the other changes in this file.

Reply:
Reverted in https://github.com/ros2/launch_ros/pull/158/commits/f66375545bf968f686c60e94fbc961f853b5109c. I will do this in a follow-up PR.